### PR TITLE
String Updates (post-1.14.7 freeze)

### PR DESCRIPTION
**Pull request by Wedge009**
Obviously not to be merged till after the 1.14.7 freeze is over (I tend to find stuff only while doing localisation updates, which happen to be during freezes...).

Happy to consider alternatives if you disagree, or even if you want to leave things as they currently stand. Just let me know constructively. ;)

Can be added to master as well, assuming no conflicts.

### CODE REVIEW

File: data/campaigns/Descent_Into_Darkness/units/Frontier_Baroness.cfg [hunk 1]
```diff
@@ -20,7 +20,7 @@
         {ABILITY_SKIRMISHER}
     [/abilities]
     usage=mixed fighter
-    description= _ "At Wesnoth’s frontiers, all able-bodied men and women must be able to defend their homes at need. Men are most often trained in the use of sword or bow and are used as soldiers who withstand the brunt of orcish assaults to hold the raiders at bay. Women, instead, learn to use the staff and sling and are most effective as scouts and ambushers. The most proficient are powerful skirmishers and are highly adept at reconnaissance, allowing them to often outmaneuver and entrap orcish raiders."+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}
+    description= _ "At Wesnoth’s frontiers, all able-bodied men and women must be able to defend their homes at times of need. Men are most often trained in the use of sword or bow and are used as soldiers who withstand the brunt of orcish assaults to hold the raiders at bay. Women, instead, learn to use the staff and sling and are most effective as scouts and ambushers. The most proficient are powerful skirmishers and are highly adept at reconnaissance, allowing them to often outmaneuver and entrap orcish raiders."+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}
```
Comment (nemaara) [inline data/campaigns/Descent_Into_Darkness/units/Frontier_Baroness.cfg]:
```suggestion
    description= _ "At Wesnoth’s frontiers, all able-bodied men and women must be able to defend their homes in times of need. Men are most often trained in the use of sword or bow and are used as soldiers who withstand the brunt of orcish assaults to hold the raiders at bay. Women, instead, learn to use the staff and sling and are most effective as scouts and ambushers. The most proficient are powerful skirmishers and are highly adept at reconnaissance, allowing them to often outmaneuver and entrap orcish raiders."+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}
```

**Comment by nemaara** [comment]
Another comment, you made a lot of changes regarding princess -> Princess, but we should actually keep https://github.com/wesnoth/wesnoth/issues/3322 in mind as well. Not sure if this PR is the place for it, but we might consider switching out the "Princess" address for something more suitable?

**Comment by Wedge009** [comment]
Have changed to 'in times of need'.

Regarding Princess, it never actually bothered me, but reading the style guide quoted in the issue, yes, that's probably because I'm thinking of the Han Solo style of irreverently addressing Leia. There seems to be bit of that kind of banter between Konrad and Li'sar and perhaps with Tallin and Eryssa too. I can understand the team wanting to make the dialogue more mature, even though I started playing Wesnoth during the 'anime-style' era.

Anyway, with respect to your question, I say we correct what we have first and if someone else wants to have a go at replacing the inappropriate uses of 'Princess', that will be handled separately as part of that issue.

File: data/campaigns/Descent_Into_Darkness/scenarios/05_Orc_War.cfg [hunk 2]
```diff
@@ -32,7 +32,7 @@
             {STORYTXT_BACKGROUND travel.jpg}
         [/part]
         [part]
-            story=_ "Malin and Darken Volk continue their raids on the orcs and eventually force the usually fractious Whitefang chieftains to unite in opposition. The orcs pursue the two necromancers east toward the mouth of the river men call the Longlier and Elves call the Arkan-thoria, where they manage to surround the humans and force a fight."
+            story=_ "Malin and Darken Volk continue their raids on the orcs and eventually force the usually fractious Whitefang chieftains to unite in opposition. The orcs pursue the two necromancers east toward the mouth of the river men call the Longlier and elves call the Arkan-thoria, where they manage to surround the humans and force a fight."
```
Comment (jostephd) [inline data/campaigns/Descent_Into_Darkness/scenarios/05_Orc_War.cfg]: [The wiki](https://wiki.wesnoth.org/NotSpellingMistakes) says:

> Remember that we capitalize the name of a race when (and only when) referring to the entire race. Thus: "the lore of the Elves", but "a band of elves".

By that rule, shouldn't this be "Elves"?

Comment (Wedge009) [reply data/campaigns/Descent_Into_Darkness/scenarios/05_Orc_War.cfg]: Ah, I missed that rule. I notice that, in general, writers seem to use capitals for non-human races, but use lower-case for referring to humans (seems rather inconsistent to me). Does Wesnoth use lower-case for humans? I note the description for woses uses lower-case.

If we are to follow this rule in the wiki, can we say '...mouth of the river Men call the Longlier and Elves call the Arkan-thoria...'?

**Comment by Wedge009** [comment]
Only change from previous state was the capitalisation issue @jostephd pointed out.

**Comment by nemaara** [comment]
Hmm, seems like the latest few commits I made introduced some conflicts in this PR. Mind if I fix them on your branch, or would you like to do it?

**Comment by Wedge009** [comment]
No worries, I was thinking to re-base when the string freeze is over anyway. I'll try to do it by the end of today (UTC+1000).

**Comment by Wedge009** [comment]
@nemaara Thanks for addressing the 'Princess addressing' issue. That was the source of most of the conflicts (as we discussed earlier I was considering it a separate issue to this PR).

* S08a: My guess is you left Tallin's address as Princess because they had only just been introduced (in the heat of battle), plus the joke is that he initially speaks informally (let's get outta...) and then belatedly remembers to speak more formally around her.
* S10a: I still capitalise Princess Eryssa, because even though Tallin is being sarcastic (due to this scene being the hostile Eryssa dialogue) he is still addressing her by her title.
* S11a: I think these instances of 'Princess' are still to be replaced with 'my lady' (I assume). Should I take of that or will you?

Also, I didn't realise 1.14.7 was released (I think I was looking at a cached copy of wesnoth.org which still linked to 1.14.6). Shall I assume the string freeze is over?

**Comment by nemaara** [comment]
S08a: I actually just missed that one. I'll fix it.
S10a: (I think you mean S09a?): sure, I can make it capitalized
S11a: I guess the "My Lady" should be capitalized too, we can just resolve those conflicts when merging (It's totally fine if you want to do it). 

String freeze is over, yes. :)

**Comment by Wedge009** [comment]
* If you think it should be removed as well, I suppose that's okay too.
* S10a:74 I only mentioned it because I already capitalised it here in this PR. Other instances of 'princess' you already removed.
* I think leaving 'my lady' as non-capital is fine - if someone was titled Lady as opposed to Princess, then it would be 'my Lady', I think.

I better merge this so you can continue you work without having to worry about conflicts. Then you can work on S11a etc.

**Comment by Wedge009** [comment]
~sigh~ WTH, I didn't mean to close this and I can't re-open it. Making a new one...

**Comment by Wedge009** [comment]
Oh heck, I'll just merge it myself. I think there were no further concerns here.

**Comment by nemaara** [comment]
Shall I cherry pick your commits to master?

**Comment by Wedge009** [comment]
I'm in the middle of doing that...

**Comment by Wedge009** [comment]
Sorry for the delay, kinda screwed up my working copy for a bit. Should be all okay now. My changes are all in both master and 1.14. I don't plan to make any further changes at this point, you can continue with your dialogue review for NR and others.

**Comment by Wedge009** [comment]
Actually, on the 'my Lady' thing, I saw some changes elsewhere of 'my Lord' to 'my lord' -  possibly with reference to Kalenz, I think.

**Comment by nemaara** [comment]
I'm not that familiar with nobility title stuff (not really a history person D: ), but I did look at a couple references: 

http://kristenstieffel.com/fiction-qa-style-royal-noble-titles/
https://merethewalther.com/2016/07/12/capitalization-youre-probably-doing-it-wrong/

Seems like it's just not capitalized, maybe.

**Comment by Wedge009** [comment]
I agree with both citations: 'my lord' and 'my lady'. So continue as you already have. The second one also makes a good point of 'highness' vs 'Your Highness'.

**Pull request closed without merging**